README.md

```diff
@@ -24,6 +24,100 @@ yarn install
 
 ```
 
+Ready your React .env:
+
+```bash
+
+cd packages/react-app
+cp .sample.env .env
+
+```
```

Comment:
I removed this part since it's not needed to run it locally. We have some defaults in the code.

E.g.

```js
export const SERVER_URL = process.env.REACT_APP_BACKEND_URL ?? "http://localhost:49832";
```
It's a good practice to provide "localhost" defaults and only use env vars to configure production / staging envs.

I'm leaving the `hardhat/.env.sample` instructions since we need the user to provide their admin address